sklearn/manifold/t_sne.py

```diff
@@ -808,17 +812,27 @@ def _tsne(self, P, degrees_of_freedom, n_samples, random_state,
         # * early exaggeration with momentum 0.5
         # * early exaggeration with momentum 0.8
         # * final optimization with momentum 0.8
+
+        # TODO sklearn 0.20: remove this warning
+        if self.learning_rate == "200 (new default)":
```

Comment:
Actually scratch that I think this is fine (I wanted to remove this comment but forgot to do so) ...